components/ui-tooling-preview/library/build.gradle.kts

```diff
@@ -0,0 +1,55 @@
+import org.jetbrains.kotlin.gradle.ExperimentalKotlinGradlePluginApi
+
+plugins {
+    kotlin("multiplatform")
+    id("maven-publish")
+    id("com.android.library")
+}
+
+val composeVersion = extra["compose.version"] as String
+
+kotlin {
+    @OptIn(ExperimentalKotlinGradlePluginApi::class)
+    targetHierarchy.default()
+    jvm("desktop")
+    androidTarget {
+        publishLibraryVariants("release")
+        compilations.all {
+            kotlinOptions {
+                jvmTarget = "11"
+            }
+        }
+    }
+    iosX64()
+    iosArm64()
+    iosSimulatorArm64()
+    js {
+        browser {
+            testTask(Action {
+                enabled = false
+            })
+        }
+    }
+    macosX64()
+    macosArm64()
+}
+
+android {
+    compileSdk = 34
+    namespace = "org.jetbrains.compose.ui.tooling.preview"
+    defaultConfig {
+        minSdk = 21
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_11
+        targetCompatibility = JavaVersion.VERSION_11
+    }
+}
+
+configureMavenPublication(
+    groupId = "org.jetbrains.compose.components",
+    artifactId = "ui-tooling-preview",
```

Comment:
```suggestion
    artifactId = "components-ui-tooling-preview",
```
- we add "components-" prefix:
<img width="384" alt="image" src="https://github.com/JetBrains/compose-multiplatform/assets/5963351/4e8135dd-f163-4605-b9dc-470c0bc7c6fa">